rfcs/0134-nix-store-layer.md

```diff
@@ -0,0 +1,257 @@
+---
+feature: nix-store-layer
+start-date: 2022-09-06
+author: John Ericson
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
```

Comment:
Thanks to the nominees for accepting! Here's the team the Steering Committee has decided on :)
```suggestion
shepherd-team: edolstra, infinisil, tomberek, L-as, andir
shepherd-leader: edolstra
```

Reply:
Hooray!